### DISCUSSION TITLE
Table of Contents not visible in blog with viewports less than 960px in width

### DISCUSSION THREAD

**Discussion by LandonTClipp** [squidfunk/mkdocs-material#5975] [Q&A] [answered]
I didn't want to make an issue because I wasn't sure if this had been discussed somewhere already. You can see in the official blog page:

https://squidfunk.github.io/mkdocs-material/blog/2022/09/12/blog-support-just-landed/

If you modify the view port to be 959px width, the TOC disappears and I can't seem to find a way to get the TOC to appear. In a typical non-blog page, the TOC would render in the hamburger menu icon on the top left (along with the other navigational sections) but the blog's navigation menu contains the categories, archive, etc, no way to drill down to the specific blog post to see the TOC.

**Comment by squidfunk** [comment, marked as answer]
Good catch! I'd consider it a bug, but the problem is that it might be tricky to solve because we have no dedicated page in the menu. You can open a new issue with a minimal reproduction and we'll look into it.

**Reply by LandonTClipp** [reply]
Still on my TODO to get an issue created. Thanks for confirming.
